doc/release-notes/upgrading.en.rst

```diff
@@ -49,6 +49,10 @@ removed in the next major release of ATS.
   * TSVConnArgIndexNameLookup
   * TSVConnArgIndexLookup
 
+* Removed Internal Libraries
+
+  * mgmt_p
```

Comment:
I don't think we need to document changes to internal libs, if end users won't notice these have been removed.